cmd/admin-replicate-status.go

```diff
@@ -326,6 +327,8 @@ func (i srStatus) String() string {
 		switch {
 		case i.MaxILMExpiryRules == 0:
 			messages = append(messages, console.Colorize("Summary", "No ILM Expiry Rules present\n"))
+		case !i.Sites[reflect.ValueOf(i.Sites).MapKeys()[0].String()].ReplicateILMExpiry:
```

Comment:
Does this code try to check if any entry in i.Sites map has ReplicateILMExpiry set to true ?

If ilm expiry is always enabled or disabled in all sites at the same time, then it is better to deprecate this field and add it to madmin.SRStatusInfo, so it will be easier to understand the logic

Reply:
When we enable ILM expiry at later stage than the enabling of site replication, the flag gets set for other sites in few seconds.
Initially we thought of but this field gets added to other site as part of replication config sync.

Reply:
@shtripat imo, this should be done on server side - report the ilm expiry replication status only if atleast one site has the feature enabled.

Reply:
Oh ok. Will move the check to server side and close this PR. Thanks!

Reply:
Sent https://github.com/minio/minio/pull/19503 to send ILM expiry replication status only if at least one site has feature enabled.